sim/battle.ts

```diff
@@ -965,6 +958,26 @@ export class Battle extends Dex.ModdedDex {
 		return null;
 	}
 
+	getAllPokemon() {
+		const qPokemon: Pokemon[] = [];
```

Comment:
why `qPokemon` and not just `pokemon`?

Reply:
Because ``pokemon`` should only be used for the type ``Pokemon``, AFAIAC.

Reply:
I'm not sure I agree, but if you feel that way, I still think `qPokemon` is an abomination that looks like wannabe Hungarian notation or something. How about `all` or `allPokemon` here and `all` or `allActive` below?

Reply:
``pokemonList`` probably?

Reply:
better than `qPokemon`, but I'm generally not a fan of encoding the type/container in the variable name. My preference is usually to just add an `'s'`, but I think we can both agree `'pokemons'` is right out :P. `allPokemon` and `allActive` seems like it would be fairly explicit given they are the return values for methods named `getAllPokemon` and `getAllActive`, no?

Reply:
I weakly prefer `pokemonList`

Reply:
I don't 'strongly' prefer `all*`, I only strongly dislike `qPokemon`. `pokemonList` works for me if you two prefer it.